### PR TITLE
br: add metrics for volume backup

### LINKED ISSUE #5533: Add metrics for br federation manager
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Metrics:
- VolumeBackup (#total, #success, #failed, total_time, backup_size).
- Cleanup (#total, #success, #failed, total_time)

Tags: namespace. Tc name, k8_cluster, fed_manager_pod_name

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by WangLe1321**
### What problem does this PR solve?

Add metrics for br federation manager

Closes #5533

### What is changed and how does it work?

Add metrics about volume backup status, volume backup total time, volume backup total size, volume backup cleanup status and volume backup cleanup total time.

For volume backup cleanup, there is no status to indicate cleanup failed. So I also add a `CleanFailed` status to backup and volume backup. If backup cleanup is failed, the backup CR will become `CleanFailed` status. If one of backup cleanup in data plane is failed, the volume backup CR will become `CleanFailed` status.

### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [x] Manual test 
- [ ] No code 

image

image

image

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: pkg/fedvolumebackup/backup/backup_manager.go [hunk 1]
```diff
@@ -183,6 +188,10 @@ func (bm *backupManager) cleanVolumeBackup(ctx context.Context, volumeBackup *v1
 			return controller.RequeueErrorf("delete backup member %s of cluster %s error: %s", backupMember.backup.Name, backupMember.k8sClusterName, err.Error())
 		}
 	}
+
+	if cleanFailed {
+		bm.setVolumeBackupCleanFailed(&volumeBackup.Status)
```
Comment (wxiaomou) [inline pkg/fedvolumebackup/backup/backup_manager.go:193]: this is very helpful~ thanks

**Review by wxiaomou — APPROVED** [review]

**Comment by wxiaomou** [comment]
@WangLe1321 @BornChanger Is there any blocker for merging this pr? Thanks

**Review by BornChanger — APPROVED** [review]

**Pull request merged**